uniffi_bindgen/src/bindings/kotlin/gen_kotlin/object.rs

```diff
@@ -20,11 +20,26 @@ impl CodeType for ObjectCodeType {
         super::KotlinCodeOracle.class_name(&self.id)
     }
 
+    fn protocol_label(&self) -> String {
+        if self.has_protocol() {
+            let mut label = self.type_label();
+            let index = self.type_label().find('?').unwrap_or(label.len());
+            label.insert_str(index, "Interface");
+            label
+        } else {
+            self.type_label()
+        }
+    }
+
     fn canonical_name(&self) -> String {
         format!("Type{}", self.id)
     }
 
     fn literal(&self, _literal: &Literal) -> String {
         unreachable!();
     }
+
+    fn has_protocol(&self) -> bool {
```

Comment:
`id` here was recently renamed to `name` to avoid confusion about what `id` actually means - so checking the name of an object being exactly "Object" doesn't seem correct?

Reply:
That's unfortunate. Thing is, we need to add the `Interface` suffix to any object that comes from the Rust SDK, but sometimes there are functions that return `java.lang.Object` and will try to also append the suffix. Is there any way to check the fully qualified name?

Reply:
I know I ran into this issue so I had to add this exception, but I can't remember where. I'll remove it and see where the tests fail.

Reply:
Oh, my mistake. It was a struct called `Object`, it was the test that needed to change.